src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Square extends Figure {
+    private static final String type = "square";
+    private final int side;
+
+    public Square(int side, String color) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's recommended to create a variable for the 'side' and 'color' before passing them to the constructor for better readability.